app/models/cinstance.rb

```diff
@@ -333,6 +333,8 @@ def to_xml(options = {})
 
       plan.to_xml(:builder => xml)
 
+      service.oidc_configuration.to_xml(builder: xml)
```

Comment:
Should we only add that when it is OIDC?

Reply:
It's cleaner IMO, but that also would mean more docs.

Reply:
OK I will do that then

Reply:
The JSON representer always display it. I will do the same